packages/components/src/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+	__experimentalText as Text,
+	__experimentalHStack as HStack,
+	Button,
+	VisuallyHidden,
+} from '@wordpress/components';
+import { useResizeObserver, useMergeRefs } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { useState, forwardRef, useRef } from 'react';
+import Menu from '../menu';
+import { BreadcrumbProps, BreadcrumbItemProps } from './types';
+import './style.scss';
+
+function BreadcrumbsMenu( { items }: { items: BreadcrumbItemProps[] } ) {
+	return (
+		<span className="breadcrumb__item-wrapper">
+			<Menu placement="bottom-start">
+				<Menu.TriggerButton
+					render={
+						<Button
+							size="compact"
+							className="breadcrumb__item"
+							text="…"
+							label={ __( 'More breadcrumb items' ) }
+						/>
+					}
+				/>
+				<Menu.Popover>
+					{ items.map( ( item, index ) => (
+						<Menu.Item
+							key={ `${ item.label }-${ index }` }
+							render={
+								<a href={ item.href } onClick={ item.onClick } className="breadcrumb__item" />
+							}
```

Comment:
We should render an `<a />` tag only if an `href` is defined

```suggestion
							onClick={ item.onClick }
							className="breadcrumb__item"
							render={
								item.href ? ( <a href={ item.href } /> ) : undefined
							}
```

Reply:
Also, is adding the `breadcrumb__item` class necessary? The styles should be already be coming from `Menu`

Reply:
Good point about the css class. Regarding the `href` it's similar to my [comment](https://github.com/Automattic/wp-calypso/pull/102998/files#r2075960246). Let's see if we have to conditionally render different tags.